documentation/modules/adding-source-provider.adoc

```diff
@@ -47,7 +46,7 @@ ifdef::rhv[]
 * *Hostname or IP address*: {manager} host name or IP address
 * *Username*: {manager} user
 * *Password*: {manager} password
-* *CA certificate*: CA certificate of the {manager}. If the {manager} Apache CA certificate was replaced with a third-party CA certificate, both CA certificates need to be specified.
+* *CA certificate*: {manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server
```

Comment:
it's not entirely accurate now - it's not the {manager} CA certificate that is being replaced by the 3rd party one, but the Apache CA certificate that is being replaced